lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,32 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(outputfileloc = pot_file_path)
```

Comment:
Not sure about the name here; `output_file_path` seems more standard than no spacing.